src/java/main/org/apache/zookeeper/metrics/Counter.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zookeeper.metrics;
+
+/**
+ * A counter refers to a value which can only increase.
+ * Usually the value is reset when the process starts.
+ */
+public interface Counter {
+
+    /**
+     * Increment the value by one.
+     */
+    public default void inc() {
+        inc(1);
+    }
+
+    /**
+     * Increment the value by a given amount.
+     *
+     * @param delta
+     */
+    public void inc(long delta);
+
+    /**
+     * Get the current value held by the counter.
+     *
+     * @return the current value
+     */
+    public long getCurrentValue();
```

Comment:
Just change it to get?

Reply:
okay, will do

Reply:
@lvfangmin  done (I did a squash commit, GitHub lost the binding to the code)